easybuild/easyconfigs/d/datamash/datamash-1.3-foss-2018a.eb

```diff
@@ -0,0 +1,24 @@
+# Author: Piotr Malicki
+
+easyblock = 'ConfigureMake'
+
+name = 'Datamash'
+version = '1.3'
+
+homepage = 'https://www.gnu.org/software/datamash/'
+description = "GNU datamash is a command-line program which performs basic numeric, textual and statistical operations on input textual data files"
```

Comment:
This line is too long, all lines should be shorter than 120 colums, so please wrap it across 2 lines:

```python
description = """GNU datamash is a command-line program which performs basic numeric, textual and
 statistical operations on input textual data files."""